src/payment/checkout/payment-form/StripePaymentForm.jsx

```diff
@@ -145,12 +147,22 @@ const StripePaymentForm = ({
         options={options}
         onReady={stripeElementsOnReady}
       />
-      <PlaceOrderButton
-        onSubmitButtonClick={onSubmitButtonClick}
-        showLoadingButton={showLoadingButton}
-        disabled={submitting}
-        isProcessing={isProcessing}
-      />
+      {isSubscription ? (
+        <SubscriptionSubmitButton
+          onSubmitButtonClick={onSubmitButtonClick}
```

Comment:
If you're planning to not have any of the functionality of this page done in this PR, can you comment out this onSubmit since it won't be that function?

Reply:
We eventually gonna change this handler I have added a TODO for now but left the `onSubmit` call there to test the error design state